ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineNewPayloadV3Test.java

```diff
@@ -55,7 +52,7 @@
 import org.mockito.junit.jupiter.MockitoExtension;
 
 @ExtendWith(MockitoExtension.class)
-public class EngineNewPayloadV3Test extends EngineNewPayloadV2Test {
+public class EngineNewPayloadV3Test extends AbstractEngineNewPayloadTest {
```

Comment:
See my suggestion above. If we stop that leakage, we should be able to still re-use the testing of prior versions.